randovania/game_connection/connector/prime_remote_connector.py

```diff
@@ -34,7 +34,7 @@ class PrimeRemoteConnector(RemoteConnector):
     version: all_prime_dol_patches.BasePrimeDolVersion
     game: GameDescription
     _last_message_size: int = 0
-    _world: Region | None = None
+    _last_emitted_region: Region | None = "PLACEHOLDER"
```

Comment:
when is `_last_emitted_region` set to `None`, now? what does it mean for it to be `"PLACEHOLDER"`?

Reply:
During `update`:
```python
            has_pending_op, region = await self.current_game_status()
            if region != self._last_emitted_region:
                self.PlayerLocationChanged.emit(PlayerLocationEvent(region, None))
            self._last_emitted_region = region
```
The idea of the change is to always emit a location changed the first time it happens, even if it's None. `GameConnection` uses that signal being emitted as a sign the RemoteConnector is not disconnected.

Though thinking about it I could make it default to `GameConnectionStatus.TitleScreen` whenever a RemoteConnector is created? hm...

Reply:
kk. in that case I just don't love seeing a magic string here - that enum entry you mentioned would be fine by mii (or whatever else you settle on)